api.md

```diff
@@ -934,7 +934,7 @@ it is not possible to chain multiple commands together in one call.
 Create the role `davide` across all data nodes in a distributed database:
 
 ```sql
-SELECT distributed_exec($$CREATE USER davide WITH PASSWORD 'jw8s0F4'$$);
+CALL distributed_exec($$CREATE USER davide WITH PASSWORD 'jw8s0F4'$$);
```

Comment:
Heads up - I think @k-rus has a separate PR which changes this.  Might want to remove this change if there are other changes to avoid the conflict.

https://github.com/timescale/docs.timescale.com-content/pull/569

Reply:
Yeah, I see he's merged that change so I'll make sure to iron out all conflicts before merging.